conf/default.toml

```diff
@@ -18,6 +18,26 @@ proxy_path = ""
 # [map]
 # max_zoom = 10
 # max_extent = [-418263.418776, 5251529.591305, 961272.067714, 6706890.609855]
+#
+# Baselayer: The default baselayer can be overridden with three properties:
+#   type: Indicates the baselayer type. Possible values are 'xyz', 'wms', 'wfs'. Mandatory if you configure a baselayer.
+#   url: Baselayer endpoint URL. Mandatory if you configure a baselayer.
+#   name: Baselayer name. Mandatory for 'wms', 'wfs' types where it indicates the layer name.
+# baselayer.type = "xyz"
+# baselayer.url = "https://{a-c}.basemaps.cartocdn.com/rastertiles/voyager/{z}/{x}/{y}.png"
+# baselayer.name = "some-layername"
+#
+# Customlayer: Besides the baselayer, addtional customlayers can be configured with the same three properties.
```

Comment:
What's not clear here is the difference between custom layers and the base layer. To me it feels like the same thing. How about simply allowing users to specify one or several default map layers, and the default config pointing to the basemap we're currently using? This is probably something you discussed cc @fgravin

Reply:
Just to be clear, the config schema is extremely important because we won't be able to change it later on except by providing a migration system, so getting the schema right is really critical IMO.

Reply:
Yes, I was thinking about simply defining the baselayer as a customlayer. It would have the drawback that we cannot check in the app anymore if a baselayer is defined in order to set a default one. What we could do in this case is to set a default layer in the app if no customlayer is defined, but this didn't feel ideal to me when the admin only wants to define a data or mask layer.
Then again, we could also always define the baselayer as the first customlayer in the config, but this would make this section of the config mandatory for the app.
Also, the separation has the advantage that the admin does not have to worry about the layer order for the baselayer, which will always be placed at the bottom.